lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -158,8 +133,6 @@ class FirstMeaningfulPaint extends Audit {
     // Parse the trace for our key events and sort them by timestamp.
     events.filter(e => {
       return e.cat.includes('blink.user_timing') ||
```

Comment:
you can fix the formatting of this filter arrow function now. i think its a one liner